app/qml/Banner.qml

```diff
@@ -77,12 +97,9 @@ Rectangle {
       source: banner.source
       textItem.font.bold: true
       textItem.rightPadding: InputStyle.innerFieldMargin
-      textItem.text: "<style>a:link { color: " + banner.linkColor
-            + "; text-decoration: underline; }</style>" +
-            qsTr("%1<br><a href='%2' style=\"color: %3;\">Learn more</a>").arg(banner.text).arg(banner.link).arg(InputStyle.learnMoreLinkColor)
+      textItem.text: banner.getText()
```

Comment:
Question: Does binding works if `getText()` is function? so if you change `banner.text` for example, does it automatically update `textItem.text`?

Reply:
Yes yes, Qt goes over `js` functions and finds bindings there. Those functions are then reevaluated when some properties used in the function change

Reply:
good to know , thanks